src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+$element-width: 166px;
+
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+    display: block;
```

Comment:
article is block element, so you don't need this style

Reply:
it is for image, not article